code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -258,6 +272,14 @@ GLOBAL_VAR(medibot_unique_id_gen)
 		show_controls(user)
 		update_icon()
 
+	if(istype(W, /obj/item/pen)&&!locked)
+		rename_bot()
+		return
+	if(istype(W,/obj/item/storage/firstaid)&&!locked)
+		var/obj/item/storage/firstaid/B = W
+		skin=B.skin_type
+		update_icon()
```

Comment:
Eh no. It's weird. Getting a colour of a box just because you tabbed it?

Reply:
I am kind of fine with this, but it could make more sense if it gives you a medical box the colour of what you replaced (if its white and you put a green medkit, then the medbot becomes green and you get a white medkit to replace your green one)

Reply:
its entirely a cosmetic change so I don't see why it should cost a box. I could replace the box interaction with a crayon interaction if that makes more sense

Reply:
yeah it makes sense more than tabbing. Closed cap of spray can would work more than crayon

Reply:
I just did a spraycan with the cap off.
Spraycans early return on their ``pre_attack()`` if their cap is on unless its in the ``spraycan_touch_normally`` list

Reply:
Make it possible when the cap is on because you would still want to spray, not reskin.

Reply:
I did some testing and code-diving and spraycans do not have any interaction with simplemobs, they do not paint them or cause any stun or anything of that sort. so unless you want me to add a way to colour a medbot using spraycans I don't see why this would be needed

Reply:
Oh yeah you're right. these are simple mobs, not objects to spray. then it will be fine.

Reply:
recolouring medkits is out of scope and I don't really see the benefit of being able to change out medkits since if you can gain one type of medkit, you can probably get any variant you want